src/masonite/facades/RateLimiter.pyi

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Callable, TYPE_CHECKING
```

Comment:
Here I discovered a way to type hint facades so that e.g. in VS Code when using facade we get completion as we were using the class directly !! It's so nice !
So in the type hinting PR, I will add those type hints for all the facades 😉

Reply:
awesome!!!!!!! that was a huge pain for me

Reply:
Do we have to create a pyi file for each facade?

Reply:
Yes..that's the counterpart. But not a big deal IMO. We are not adding facades every day, and we will just need to take care during code review of new features, that a facade (if necessary) exists and that its `pyi` counterpart exists too.

It's a documentation on its own so it's as important as maintaining doc on the site IMO.